bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/BulkRead.java

```diff
@@ -100,10 +104,12 @@ public BulkRead(AsyncExecutor asyncExecutor, String tableName) {
    */
   public void flush() throws InterruptedException {
     if (futures != null && !futures.isEmpty()) {
+      LOG.info("BulkRead reading %d rows.", futures.keys().size());
```

Comment:
I'm fine having it in info for now to help development of bulk api, but it should eventually be debug or gone.